tstyche.config.json

```diff
@@ -0,0 +1,4 @@
+{
+  "$schema": "https://tstyche.org/schemas/config.json",
+  "testFileMatch": ["src/**/__tests__/*.test-tsd.ts"]
```

Comment:
I left `tsd` in the file name to keep the diff readable. Renaming the file to `*.tst.ts` will allow removing the config file in the future.